app/shared/accessibility-shortcuts/_accessibility-shortcuts.scss

```diff
@@ -0,0 +1,18 @@
+.accessibility-shortcuts {
+  text-align: center;
+  background-color: $theme-info-light;
+  
+  .skip-link {
+    &:focus,
+    &:active {
+      position: relative;
+      left: 0;
```

Comment:
Left property on a non absolute position?

Reply:
I actually just copy pasted these rules. I'll try and validate that it doesn't reset anything.

Reply:
So, I think this is what I've done here:  
1) I copied over the general logic for hiding elements from vero.fi
2) I copied the logic for showing elements from Turku's version of Varaamo

This created a miss match between the styles for displaying and hiding elements.

I preferred Vero.fi's approach, because they were using `sr-only`. Meaning that they are showing the content to screen readers even when it's not focused (Vero.fi is also using Bootstrap). Turku's approach may also achieve this, but the rules they were using were custom.

I've now copied the rules for "resetting" `sr-only` from vero.fi.